code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -519,3 +519,46 @@ Technomancer RIG
 		/obj/item/rig_module/ai_container,
 		/obj/item/rig_module/storage/large
 		)
+
+/***************************************
+	MU66 Suit
+****************************************/
+/obj/item/rig/sea
+	name = "MU-66 Atmos suit"
+	suit_type = "MU66 Soft-suit"
+	desc = "A light RIG for performing minute repairs and maintenance in atypical atmospheres."
+	icon_state = "sea_rig"
+	armor_list = list(
+		melee = 40,
+		bullet = 30,
+		energy = 20,
+		bomb = 10,
+		bio = 100,
+		rad = 100
+	)
+	seal_delay = 20
+	slowdown = 0
+	offline_slowdown = 1
+	offline_vision_restriction = 1
+	removable_cell = TRUE
+	cell_type =  /obj/item/cell/large/moebius/nuclear
```

Comment:
suit includes a large atom cell...? Seems kinda bad for balance

Reply:
man, I kinda overlooked the armor things on accident. I forgot I made this before the new armor PR and just wanted to rush it through to get things off my plate. I'll get the issues changed up real quick